02-documentation/developer-guides/building-a-nextjs-blog-with-squidex.md

```diff
@@ -0,0 +1,588 @@
+---
+description: >-
+  How to build a Next.js blog powered by content from Squidex CMS.
+---
+
+# Building a blog with Squidex and Next.js
+
+### Introduction
+
+This tutorial will guide you to create a blog application, set up its schema, and add mock content on Squidex, and create a Next.js blog based on the data stored in your CMS.
```

Comment:
"and" should be the last separator in an enumeration.

like a, b, c and d NOT a, b and c and d